logic/src/commonMain/kotlin/com/wire/kalium/logic/util/Base64.kt

```diff
@@ -0,0 +1,10 @@
+package com.wire.kalium.logic.util
+
+/**
+ *
+ * TODO: Move to a utils module?
```

Comment:
lol, see my first comment 😅

Reply:
Yep.
I think some Utils would be cool.

Base64, Either, UUID, etc. could go there.